### ISSUE TITLE
Run 3 crashes the desktop app

### ISSUE THREAD

**Issue by n0samu** [ruffle-rs/ruffle#7019]
### Describe the bug

When opening the [Run 3 SWF](https://archive.org/download/a-collection-of-flash-games-swf/Run%203.swf) with the Ruffle desktop app on Windows, it briefly shows a black screen and then crashes. Even though the SWF uses ActionScript 3, no warning is displayed. The output from the Command Prompt is below:
```
thread 'main' panicked at 'Error in Queue::write_texture: copy of 0..100464 would end up overrunning the bounds of the Source buffer of size 66976', C:\Users\runneradmin\.cargo\registry\src\github.com-1ecc6299db9ec823\wgpu-0.12.0\src\backend\direct.rs:214:9
```

### Expected behavior

This SWF uses ActionScript 3, so the "This content is not yet supported by Ruffle" warning box should be displayed before running the SWF. And ideally a crash would not occur.

### Affected platform

Desktop app

### Operating system

Windows 10

**Comment by Herschel** [comment]
This will be fixed by #7011.

**Comment by Herschel** [comment]
Panic fixed by #7011.
